content/advanced/430_emr_on_eks/autoscaling.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Configure Autoscaling"
+date: 2021-04-08T01:29:46-07:00
+weight: 40
+draft: false
+---
+
+Autoscaling in EKS is achieved using Cluster Autoscaler.
+
+When resources are not available EKS will add new nodes to the cluster to create new pods and execute the jobs.
+
+Refer to [Cluster autoscaler](beginner/080_scaling/deploy_ca/) section in order to configure the cluster autoscaler.
```

Comment:
Let's aim to give complete experience for workshop users. Be creative, think about simple and complex use case and think about what an analytics person should know about autoscaling. We need to add another chapter about DRA (dynamic resource allocation) as well

Reply:
Nit: Should clarify that this is vertical autoscaling?